subworkflows/local/eukulele.nf

```diff
@@ -28,7 +29,15 @@ workflow SUB_EUKULELE {
         EUKULELE_SEARCH( ch_eukulele )
 
         FORMAT_TAX( EUKULELE_SEARCH.out.taxonomy_estimation.map { [ it[0], it[1] ] } )
-        SUM_TAXONOMY( FORMAT_TAX.out.tax, collect_fcs )
+
+        FORMAT_TAX.out.tax
+            .join(ch_eukulele)
+            .map { [ it[0], it[3], it[1] ] }
+            .set { ch_sum_taxonomy }
+        ch_sum_taxonomy.view()
+        feature_counts.view()
```

Comment:
Should these really be in here? I don't see any issue with `.view`, it's just that I've mostly used it to verify code.